bin/opm

```diff
@@ -183,30 +197,29 @@ sub do_build ($) {
 
     my $account;
     if (!$server_build) {
-        my ($rcfile, $data) = get_rc_file();
-        my $default_sec = $data->{default};
-        $account = delete $default_sec->{github_account};
+        my ($rc_file, $rc_data) = get_rc_file();
+        my $rc_default_sec = $rc_data->{default};
```

Comment:
Please avoid such unrelated changes. I prefer minimal change sets in each PR. Thank you.